src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -304,6 +321,43 @@ func startNode(
 	return err
 }
 
+func checkBrokers(nodes []node) func() error {
+	return func() error {
+		addrs := make([]string, 0, len(nodes))
+		for _, n := range nodes {
+			addrs = append(addrs, n.addr)
+		}
+		client, err := kafka.InitClient(addrs...)
+		if err != nil {
+			return err
+		}
+		lenBrokers := len(client.Brokers())
+		if lenBrokers != len(nodes) {
+			return fmt.Errorf(
+				"Expected %d nodes, got %d.",
+				len(nodes),
+				lenBrokers,
+			)
+		}
+		return nil
+	}
+}
+
+func waitForCluster(check func() error) error {
+	retries := uint(3)
```

Comment:
should we print something to the user? Like "Waiting for cluster to be ready..." ?

Reply:
That sounds good.